x-pack/plugins/infra/public/components/source_configuration/indices_configuration_panel.tsx

```diff
@@ -43,101 +39,51 @@ export const IndicesConfigurationPanel = ({
       </h3>
     </EuiTitle>
     <EuiSpacer size="m" />
-    {displaySettings === 'metrics' && (
-      <EuiDescribedFormGroup
-        title={
-          <h4>
-            <FormattedMessage
-              id="xpack.infra.sourceConfiguration.metricIndicesTitle"
-              defaultMessage="Metric indices"
-            />
-          </h4>
-        }
-        description={
+    <EuiDescribedFormGroup
+      title={
+        <h4>
           <FormattedMessage
-            id="xpack.infra.sourceConfiguration.metricIndicesDescription"
-            defaultMessage="Index pattern for matching indices that contain Metricbeat data"
+            id="xpack.infra.sourceConfiguration.metricIndicesTitle"
+            defaultMessage="Metric indices"
```

Comment:
[nit] ...maybe? Somehow it sounds better in my head. The current name makes me want to add an "Imperial indices" setting with all metrics in miles and yards and feet and Farenheit.

```suggestion
            defaultMessage="Metrics indices"
```

Reply:
Yes, I see your point. I changed it to `Metrics`.